apps/internal/tasks.py

```
@@ -40,10 +42,10 @@ def re_export_stuck_exports() -> None:
         for orm in ormqs:
             if 'chain' in orm.task and orm.task['chain']:
                 for chain in orm.task['chain']:
-                    if len(chain) > 1 and isinstance(chain[1], list) and isinstance(chain[1][0], ExpenseGroup):
-                        if chain[1][0].id in expense_group_ids:
-                            logger.info('Skipping Re Export For Expense group %s', chain[1][0].id)
-                            expense_group_ids.remove(chain[1][0].id)
+                    if len(chain) > 1 and chain[0] in target_func and isinstance(chain[1][0], int):
+                        if chain[1][0] in expense_group_ids:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)